vite.config.js

```diff
@@ -15,9 +15,9 @@ import viteImagemin from "vite-plugin-imagemin";
 import vue from "@vitejs/plugin-vue";
 import Pages from "vite-plugin-pages";
 import generateSitemap from "vite-plugin-pages-sitemap";
-import "jquery";
-import $ from " jquery";
-import jQuery from "jquery";
+// import "jquery";
+// import $ from " jquery";
```

Comment:
Why remove the aliases?

Reply:
Impossible to start de dev server (or build for prod) otherwise
```
Error: Cannot find module ' jquery'
Require stack:
- /app/vite.config.js
```

Reply:
ho my bad there is a typo !